NC Kilderkin - bar/brewery ship

ErhardSteinhauer opened the pull request:
## About the PR
_"Spaceworthy bar/microbrewery with everything one needs to facilitate poor life choices: lots of booze, smokes, and lack of food."_, - Shipyard.
**Ship size:** Medium (264 tiles)
**Recommended crew:** 3
**Cost:** 42900
**Power Gen Type:** Plasma
**Expeditions:** No
**IFF Console:** No
**Available Roles:** Bartender, Botanist, Mercenary, Pilot

## Why / Balance
Dedicated bar ship. NSFD better start their printer for tickets for drunken driving.

## Technical details
.ymls

## Media
![2024-1-22_20 55 15](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/c1734cb8-d076-489e-95c8-a09e143b105d)
![2024-1-22_20 56 40](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/838bc18c-8048-4e85-9ffd-6ff31fbfd642)
- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl: erhardsteinhauer
- add: Added NC Kilderkin - bar/brewery ship.

TsjipTsjip reviewed (approved):
Facts:
- Ship price at shipyard: 42900
- Ship sale value at shipyard: 38754

Minor notes:
- Musician's corner is a bit dark. Moody tho, could argue keeping it like this.

This ship is a good example of a bar ship that is quite literally nothing else except for a little bit of botany, but it does not have a kitchen which keeps it true to its name. I see no issues with this ship as of now.

ErhardSteinhauer commented:
> Facts:
> 
> * Ship price at shipyard: 42900
> * Ship sale value at shipyard: 38754

I think I went for 10% markup.

> Minor notes:
> 
> * Musician's corner is a bit dark. Moody tho, could argue keeping it like this.

There is a lamp in that corner for those who'd like more light.

>This ship is a good example of a bar ship that is quite literally nothing else except for a little bit of botany, but it does not have a kitchen which keeps it true to its name. I see no issues with this ship as of now.

Debated if I even should have botany there, but some cocktails require fruits or veggies.

dvir001 commented:
Ship mark up need to be 6% up to 15%
I think McCargo is 15%
Loader is 7%
And so on

ErhardSteinhauer commented:
> Facts:
> 
> * Ship price at shipyard: 42900
> * Ship sale value at shipyard: 38754

Wait, do you mean despite me adding 10% markup it sells without one?

FoxxoTrystan commented on Resources/Prototypes/_NF/Shipyard/kilderkin.yml:
```diff
@@ -0,0 +1,29 @@
+
+- type: gameMap
+  id: kilderkin
+  mapName: 'NC Kilderkin'
```
Remove "NC" from the napName as NC should not appear in the IFF Tag.

ErhardSteinhauer replied:
Done

FoxxoTrystan commented on Resources/Prototypes/_NF/Shipyard/kilderkin.yml:
```diff
@@ -0,0 +1,29 @@
+  minPlayers: 0
+  stations:
+    kilderkin:
+      stationProto: StandardFrontierExpeditionVessel
```
StandardFrontierExpeditionVessel is for Expedition ships, use StandardFrontierVessel.

ErhardSteinhauer replied:
And done

ErhardSteinhauer commented:
Fixed shipyard file for the ship.

Cheackraze commented:
lgtm. The engi locker is a little sus but as long as its not a hard suit locker, this looks just fine. I'm also ok with it having only a 10% markup since there isnt really anything on the ship to make it like, some kind of money printing ship.

ErhardSteinhauer commented:
I think it's just engilocker without the hardsuit (hence the emergency wall locker in the same room). It's mostly there for looks. And for filled utility belt.

ErhardSteinhauer commented:
Updated the ship to current standards and images in PR:
- Enabled Pilot Job.
- Replaced Engineer Locker with Utility Belt [Filled]
- Replaced Emergency EVA Wall Closet with Suit Wallstorage Unit (EVA).
- Swapped some consoles for tabletop versions.

MagnusCrowe commented:
> Updated the ship to current standards and updated the images in PR:
> 
> * Enabled Pilot Job.
> * Replaced Engineer Locker with Utility Belt [Filled]
> * Replaced Emergency EVA Wall Closet with Suit Wallstorage Unit (EVA).
> * Swapped some consoles for tabletop versions.

Are you sure?  I just loaded it up and the pilot job isn't on the records computer.

Ship looks good.  I need to check with Checkraze about the camera server though, I'll get back to you.

ErhardSteinhauer commented:
I'm fairly sure, yeah.
![2024-1-23_08 08 57](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/fd576a36-b462-4431-b90e-e989137a129b)

MagnusCrowe commented:
> I'm fairly sure, yeah. 

Wild, I could have sworn I only saw 3 options last time I booted it up.  I see it now...I must be going insane.

No issues with the camera server.  Will probably be approving it today.
